templatetags/htk_tags.py

```diff
@@ -302,6 +302,23 @@ def get_request_duration():
     return duration
 
 
+@register.simple_tag()
+def localize(key):
+    '''TODO: handle this based on `by_language`
```

Comment:
Docstrings should always be surrounded by triple quotes

See https://peps.python.org/pep-0008/